src/components/battleItemContainer.html

```diff
@@ -44,16 +46,76 @@
+          Flutes are disabled.
+      </p>
+      <!-- Flutes -->
+      <table class="table table-bordered m-0" data-bind="visible: App.game.badgeCase.hasBadge(BadgeEnums.Heat)">
+        <tbody>
+          <tbody data-bind="foreach: GameHelper.chunk(6, Object.keys(ItemList).filter(i=>ItemList[i].constructor.name == 'FluteItem'))">
+            <tr data-bind="foreach: $data">
+                 <td class="amount p-0" data-bind="text: GameHelper.formatAmount(player.itemList[$data]()),tooltip:{title:player.itemList[$data](),trigger:'hover',placement:'top',animation:false}"></td>
+            </tr>
+            <tr data-bind="foreach: $data">
+                 <td class="p-0" data-bind="attr: {
+                                disabled: !player.itemList[$data]()
+                            },
+                            css: {
+                                'bg-primary': fluteEffectRunner.getEffect($data) > 30,
+                                'bg-warning': fluteEffectRunner.getEffect($data) <= 30 && fluteEffectRunner.getEffect($data) > 0,
+                                'bg-secondary': !fluteEffectRunner.isActive($data)()
+                            },
+                            event: {
+                                click: function(){ItemHandler.useItem($data)}
+                            },
+                            tooltip: {
+                              title: ItemList[$data].displayName + '<br/>' + ItemList[$data].description + '<br/>' + 'Utilizes: ' + ItemList[$data].gemTypes[0] + ', ' + ItemList[$data].gemTypes[1] + ', and ' + ItemList[$data].gemTypes[2] + ' Gems',
+                              trigger: 'hover',
+                              placement:'bottom',
+                              html: true
+                            }">
+                <img src="" class="clickable" data-bind="attr: { src: ItemList[$data].image }" width="26px">
 
+            </td>
+            </tr>
+            <tr data-bind="foreach: $data">
+                <td class="text-light p-0">
+              <!-- ko if: fluteEffectRunner.getEffect($data) > 0 -->
+                <code data-bind="
+                text: player.effectTimer[$data](),
+                tooltip: {
+                  title: 'Gems/Second: ' + fluteEffectRunner.numActiveFlutes() +
+                     '<br><br>Shard Types Used: <br>' +
+                     ItemList[$data].gemTypes[0] + ': ' + App.game.gems.gemWallet[PokemonType[ItemList[$data].gemTypes[0]]]() + '<br>' +
+                     ItemList[$data].gemTypes[1] + ': ' + App.game.gems.gemWallet[PokemonType[ItemList[$data].gemTypes[1]]]() + '<br>' +
+                     ItemList[$data].gemTypes[2] + ': ' + App.game.gems.gemWallet[PokemonType[ItemList[$data].gemTypes[2]]]() + '<br>' +
+                     '<br> Time Remaining: <br>' + GameConstants.formatSecondsToTime(player.effectList[$data]()),
```

Comment:
Might want all of this in a function on fluteEffectRunner instead, to keep the html simple.